From de77cf5eea58c3ef991fb5b897c1517f6c528006 Mon Sep 17 00:00:00 2001
From: Andreas Larsson <andreas@gaisler.com>
Date: Tue, 15 Sep 2020 13:48:33 +0200
Subject: [PATCH 28/50] common: Add defines for when TN-0010 and/or TN-0011 are
 enabled

Although UT699 and UT700 technically are affected by TN-0011, we do not
fix this elsewhere in GCC for the single core UT699 and UT700.
---
 gcc/config/sparc/sparc-c.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/gcc/config/sparc/sparc-c.c b/gcc/config/sparc/sparc-c.c
index ec6cb29b049..c95558b6e2d 100644
--- a/gcc/config/sparc/sparc-c.c
+++ b/gcc/config/sparc/sparc-c.c
@@ -42,6 +42,10 @@ sparc_target_macros (void)
 
   if (sparc_fix_b2bst)
     cpp_define (parse_in, "__FIX_LEON3FT_B2BST");
+  if (sparc_fix_ut700)
+    cpp_define (parse_in, "__FIX_LEON3FT_TN0010");
+  if (sparc_fix_gr712rc)
+    cpp_define (parse_in, "__FIX_LEON3FT_TN0011");
 
   if (TARGET_ARCH64)
     {
-- 
2.34.1

